pages/1.10/administering-clusters/replacing-a-master-node/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: layout.pug
+navigationTitle:  Replace a master node
+title: Replace a master node
+menuWeight: 800
+excerpt:
+
+enterprise: false
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-docs-site -->
+
+
+You can replace a master node in an existing DC/OS cluster. Only one master should be replaced at a time.
+
+### Prerequisites:
+
+*   The DC/OS cluster has 3 or 5 master nodes.
+*   DC/OS is installed using the [custom](/1.10/installing/oss/custom/) installation method.
+*   The archived DC/OS installer file (`dcos-install.tar`) from your [installation](/1.10/installing/oss/custom/gui/#backup).
```

Comment:
I think we are deprecating the GUI installer and the latest version of DC/OS (1.11) does not even have documentation for how to use GUI installer. It would be best. if we not refer to something thats going away. Perhaps we can move the backup instructions to else where in the docs and point to that  or just tell the customers to backup/use contents of their `genconf/serve` directory.

Reply:
I'm not sure I fully understood what you are saying. But if you are saying lets move https://docs.mesosphere.com/1.10/installing/oss/custom/gui/#optional-backup-your-dcos-installer-files to Advanced installer docs and reference that. Sounds like a good idea to me as that procedure is not GUI specific

Reply:
Wow. I re-read my comment and TBH I don't know what I meant either.